doc/book/service-manager.md

```diff
@@ -0,0 +1,99 @@
+# Service Manager
+
+The `zend-log` package provides several components which can be used in 
+combination with the service manager. These components make it possible
+to quickly setup a logger instance or to provide custom writers, filters,
+formatters or processors.
+
+## LoggerAbstractServiceFactory
+
+When you register the abstract factory called `Zend\Log\LoggerAbstractServiceFactory`,
+you will be able to setup loggers via the configuration. The abstract factory can be
+registered in the service manager like this:
+
+```php
+// module.config.php
+return [
+    'service_manager' => [
+        'abstract_factories' => [
+            'Zend\Log\LoggerAbstractServiceFactory',
+        ],
+    ],
+];
+```
```

Comment:
I can make that change on merge.

@waltertamboer What @samsonasik is saying is that the registration of the abstract factory is only necessary in ZF2 applications; once you bump to v3 and/or start using zend-component-installer, the registration happens automatically for you. What I'll do on merge is add version-specific verbiage.